content/_help/verify-your-identity/verify-your-identity-in-person._en.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Verify your identity in person
+category: verify-your-identity
+permalink: /help/verify-your-identity/verify-your-identity-in-person/
+order: 7
+---
+
+If you are unable to successfully upload your ID on Login.gov, you may be able to verify in person with our partners at the United States Postal Service (USPS). At this time, we’re only offering this option at [select USPS locations in the Washington D.C. area](#available-locations-in-the-washington-dc-area) and for certain partner agencies.
+
+## Start this process on Login.gov before going to the Post Office 
+
+Please have the following information ready to enter on Login.gov:
+
+1. Your State-Issued ID. [Don’t have a state issued ID?](/help/verify-your-identity/accepted-state-issued-identification/)
+2. Social Security Number
+3. A phone number on a phone plan that is in your name
+   * If you do not have a phone plan that is in your name, we can send you the verification code by mail which takes approximately 3-5 days.
+
+If you’re unable to complete any of these steps, please contact the partner agency’s help center for support.   
+
+## Bring the following items to the Post Office
+
+### A copy of your emailed barcode
+
+Once you complete the steps on Login.gov, we’ll email you a barcode and a deadline to go to the Post Office. You can print the email, or show the email from your mobile device. If you go to the Post Office after the deadline, your information will not be saved and you will need to restart the process.  
+
+### Your state-issued ID
+
+Bring the same ID you used on Login.gov. Your ID must not be expired. At this time, only the following state-issued identification is accepted: 
+
+* Driver’s license from all 50 states, the District of Columbia (DC), and other US territories (Guam, US Virgin Islands, Mariana Islands and Puerto Rico)
+* A non-driver’s license state-issued ID card
+  * This is an identity document issued by the state, the District of Columbia (DC), or US territory that asserts identity but does not give driving privileges.
+
+We do not currently accept any other forms of identification, such as passports and military IDs. You cannot verify your identity at the Post Office without a state-issued ID. We’re currently working to add more ways to verify your identity.
+
+In the meantime, please contact the partner agency’s help center to find out what you can do.
+
+### Proof of your current address
+
+If your current address does not match the address on your ID, you will need to bring proof of your current address to the Post Office. Acceptable forms of proof of address are:
+
+* Lease, Mortgage, or Deed of Trust
+* Voter Registration
+* Vehicle Registration Card
+* Home or Vehicle Insurance Policy
+
+You can bring a physical copy or show a digital copy of the document. You cannot show a photo or screenshot of the document. 
```

Comment:
@aduth, Lalitha is still waiting for a response from privacy on this. If we don't hear back in time, we may need to go live with this edit:

> You must bring a physical copy of the document. You cannot show a photo or screenshot of the document.

Reply:
Preparing suggestion in case we need to swap it out:

```suggestion
You must bring a physical copy of the document. You cannot show a photo or screenshot of the document.
```

Reply:
Confirmation at: https://gsa-tts.slack.com/archives/C03FA4VBN76/p1660669598402609